api/build.gradle

```diff
@@ -22,8 +22,8 @@ plugins {
 
 archivesBaseName = 'stellarstation-api'
 
-sourceCompatibility = '1.7'
-targetCompatibility = '1.7'
+sourceCompatibility = '1.8'
```

Comment:
Just curious did anything in curiostack require this bump?

Reply:
Leaving source compatibility at 1.7 is fine, but target needed a bump, looks like it is due to checker-qual.

```
> No matching variant of org.checkerframework:checker-qual:3.10.0 was found. The consumer was configured to find an API of a library compatible with Java 7, preferably in the form of class files, and its dependencies declared externally but:
       - Variant 'apiElements' capability org.checkerframework:checker-qual:3.10.0 declares an API of a library, packaged as a jar, and its dependencies declared externally:
           - Incompatible because this component declares a component compatible with Java 8 and the consumer needed a component compatible with Java 7
       - Variant 'javadocElements' capability org.checkerframework:checker-qual:3.10.0 declares a runtime of a component, and its dependencies declared externally:
           - Incompatible because this component declares documentation and the consumer needed a library
           - Other compatible attributes:
               - Doesn't say anything about its target Java version (required compatibility with Java 7)
               - Doesn't say anything about its elements (required them preferably in the form of class files)
       - Variant 'runtimeElements' capability org.checkerframework:checker-qual:3.10.0 declares a runtime of a library, packaged as a jar, and its dependencies declared externally:
           - Incompatible because this component declares a component compatible with Java 8 and the consumer needed a component compatible with Java 7
       - Variant 'sourcesElements' capability org.checkerframework:checker-qual:3.10.0 declares a runtime of a component, and its dependencies declared externally:
           - Incompatible because this component declares documentation and the consumer needed a library
           - Other compatible attributes:
               - Doesn't say anything about its target Java version (required compatibility with Java 7)
               - Doesn't say anything about its elements (required them preferably in the form of class files)
```